### ISSUE TITLE
The name of the function `nn.L1Loss()` should be `nn.MAE()` or the name of the function `MSELoss()` should be `nn.L2Loss()`

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#128779]
### 🚀 The feature, motivation and pitch

The name of the function [nn.L1Loss()](https://pytorch.org/docs/stable/generated/torch.nn.L1Loss.html#torch.nn.L1Loss) should be `nn.MAE()` or the name of the function [MSELoss()](https://pytorch.org/docs/stable/generated/torch.nn.MSELoss.html#torch.nn.MSELoss) should be `nn.L2Loss()` so that we can easily know these functions are similar.

**Comment by albanD** [comment]
Changing these names would obviusly be very much BC-breaking. So not very feasible.
Also searching any of these on google will point to the appropriate doc page: looking for "pytorch MAE loss" points to our L1Loss page and same for "pytorch L2 loss" points to our MSELoss page.

Not sure what can be done here?
